Marlin/Conditionals_post.h

```diff
@@ -242,9 +242,9 @@
     #define MAX6675_TMAX 1024
   #elif TEMP_SENSOR_0 == -1
     #define HEATER_0_USES_AD595
-  #elif TEMP_SENSOR_0 == 0
-    #undef HEATER_0_MINTEMP
-    #undef HEATER_0_MAXTEMP
+//  #elif TEMP_SENSOR_0 == 0
+//    #undef HEATER_0_MINTEMP
+//    #undef HEATER_0_MAXTEMP
```

Comment:
Instead of commenting out, simply excise. Git will remember what was there. If commenting out is useful to help document alternative ideas, comment out at the indentation level rather than at the left margin.